### ISSUE TITLE
Many zip codes missing

### ISSUE THREAD

**Issue by phonon-zz** [petewarden/openheatmap#11]
There are many zip codes missing from http://www.openheatmap.com/locationvalueshelp.php?name=zip_code_area

For example, 80129 is missing, Littleton, CO
http://www.zip-codes.com/zip-code/80129/zip-code-80129.asp

Is this an oversight? Can it be fixed or is there a workaround? Otherwise, this is not robust enough for national US mapping...Thanks!

**Comment by petewarden** [comment]
Unfortunately there are a decent number of ZIP codes missing from the only public source of their boundaries, the US Census:
http://www.census.gov/geo/www/cob/z52000.html
I have some ideas on how to create a better open map of ZIPs, but until then it is tough to use for choropleth/colored area maps.
One alternative that may work for you is more of a point/circle based map, which does work if you put a column header of 'location' instead of 'zip_code_area' in your source spreadsheet.

**Comment by phonon-zz** [comment]
Hmmm...you may want to try the updated 2010 data sets
ftp://ftp2.census.gov/geo/tiger/TIGER2010/ZCTA5/2010/

has the 2010 ZIP (technically ZCTA5) shapefiles.

**Comment by chrisspen** [comment]
Stumbled across this ticket while researching the same problem involving django-census-places. I extended that project to load the 2010 zip code boundaries. Sadly, that data set also seems to be missing thousands of boundary definitions.

**Comment by Jmuccigr** [comment]
Another missing one is 06824 from Fairfield, CT.

**Comment by chrisspen** [comment]
I emailed geo.tiger@census.gov and they responded that some ZIPs won't have ZCTA data if they only represent P.O. boxes. So some missing data is expected.

**Comment by Jmuccigr** [comment]
Yes, but my example wasn't one. This is a house address that has it.
